rasp/tor_engine.py

```diff
@@ -1,57 +1,63 @@
 import getpass
-import http.client
-import time
-import urllib
-import urllib.error
-import urllib.request
+import os
 
-import stem.connection
 from stem import Signal
+from stem.control import Controller
 
-from rasp.base import Webpage, DefaultEngine
+from rasp.base import DefaultEngine
 
 
 class TorEngine(DefaultEngine):
-    def __init__(self, pw=None, control=None, signal=Signal.NEWNYM, proxy_handler=None, data=None, headers=None):
-        super(TorEngine, self).__init__(data, headers)
-
-        self.signal = signal
-        self.pw = pw or getpass.getpass("Tor password: ")
-        self.control = control or ("127.0.0.1", 9051)
-
-        default_handler = urllib.request.ProxyHandler({"http": "127.0.0.1:8118"})
-        self.proxy_handler = proxy_handler or default_handler
+    def __init__(self,
```

Comment:
I like the option for environment variables here, but we need to document that.  Could you add a sphinx-style docstring explaining the order of defaults and what they are for these input params?

If you haven't written a docstring before it's something like: 

```
"""short description

Long description.

:param param1: description
...
:param N: description
"""
```